addons/kubearmor/enable

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+CURRENT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+
+curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b "$SNAP_COMMON/bin"
```

Comment:
How do you make sure you install the version: "0.10.2" and not the latest?

Reply:
`karmor install` will by default install the [current STABLE_RELEASE](https://github.com/kubearmor/KubeArmor/blob/main/STABLE-RELEASE) for kubearmor.

Reply:
@webdevgopi , if we want to install v0.10.2 specifically ... we should be using `karmor install --image=kubearmor/kubearmor:v0.10.2`

Reply:
Hi,
Updated the install command with `karmor install --image=kubearmor/kubearmor:v0.10.2`

Reply:
Thank you for this fix. Our preference is for addons to deploy specific versions and not the latest stable of a product because the addon should not be affected by breaking changes. For example lets say in a year kubearmor ships a version that is no more compatible with k8s 1.28 resulting in this addon to break blocking our CI. If you need to update the kubearmor version on a MicroK8s release you will need to submit a PR against the respective branch in this repository.